app/src/main/kotlin/com/mapbox/vision/teaser/replayer/ArReplayNavigationActivity.kt

```diff
@@ -82,24 +82,33 @@ class ArReplayNavigationActivity : AppCompatActivity(), MapboxMap.OnMapClickList
     private var navigationMap: NavigationMapboxMap? = null
     private var navigation: MapboxNavigation? = null
     private var lastLocation: Location? = null
-    private var mapVisible = false
+    private var toolsVisible = false
 
     private lateinit var sessionPath: String
     private lateinit var mapView: MapView
     private lateinit var destination: Point
 
     private var activeArFeature: ArFeature = ArFeature.LaneAndFence
+    private var playbackTracked = false
 
     private val visionListener = object : VisionEventsListener {
         override fun onVehicleStateUpdated(vehicleState: VehicleState) {
             runOnUiThread {
+                if (playbackTracked) {
```

Comment:
just a question:
should you add `if (visionManagerMode == Replay` here?

Reply:
This ArReplayNavigationActivity is created only in Replay mode